src/scripts/lexmatch-sssom-compare.py

```diff
@@ -204,7 +204,7 @@ def extract_unmapped_matches(input: str, matches: TextIO, output_dir: str, summa
     combined_df = pd.concat(ont_df_list)
 
     combined_msdf = MappingSetDataFrame(
-        df=combined_df, prefix_map=msdf_lex.prefix_map, metadata=msdf_lex.metadata
+        df=combined_df, converter=msdf_lex.converter, metadata=msdf_lex.metadata
```

Comment:
## `preifx_map` -> `converter`
This is good; allows us to use the EPM (inside of `converter`) instead.

Reply:
afaics `prefix_map` does not even exist anymore in sssom-py.